app/models/manageiq/providers/redhat/infra_manager/host.rb

```diff
@@ -1,4 +1,6 @@
 class ManageIQ::Providers::Redhat::InfraManager::Host < ::Host
+  MINIMUM_CONVERSION_HOST_VERSION = '4.2.4'.freeze
```

Comment:
A constant just used in one place seems like overkill, think its fine to just use `"4.2.4"` below with a comment about this being the minimum supported version (thought the unsupported_reason_add is pretty self explanatory even)

Reply:
Another option would be to put this in settings.yml, but I'm fine with whatever other folks prefer.

Reply:
Even more overkill, don't think we'd ever want to allow that to be changed right?  I was only added to rhev once.

Reply:
I was thinking this would let people switch to `4.2.31` if desired, but if that's a bad idea, then I'll just hard code it.

Reply:
@agrare I was just trying to provide maximum flexibility as per @fdupont-redhat's comment at https://github.com/ManageIQ/manageiq-providers-ovirt/pull/315#issuecomment-444371061